lib/apiservers/service/swagger.json

```diff
@@ -661,7 +661,15 @@
                 "shares": { "$ref": "#/definitions/Shares" }
               }
             },
-            "resource": { "$ref": "#/definitions/Managed_Object" }
+            "resource": { "$ref": "#/definitions/Managed_Object" },
+            "affinity": {
+              "type": "object",
+              "properties": {
+                "use_vm_group": {
+                  "type": "boolean"
```

Comment:
I am inclined towards enums rather than named booleans. Whatever's most in line with the rest of the API.

Reply:
What would you expect the other enum values to be?